.travis.yml

```diff
@@ -3,6 +3,7 @@ python:
 #  - 3.5
   - 3.6
   - 3.7
+  - 3.8
```

Comment:
Cool. This could be split into its own PR since it is unrelated to the RTD work. Good engineering practice is to scope pull requests to one key concept. This makes it easier for reviewers to do a review quickly and makes the code more maintainable long term. If in the future you need to troubleshoot a bug using `git bisect` (not used often but when you need it you will be glad it exists), smaller well scoped PRs make bugs easier to find.